### ISSUE TITLE
video_transfer_function issue

### ISSUE THREAD

**Issue by Pinoginooo** [defl/directshow_metadata_injector_filter#1]
I have tried your filter, but I had to disable it because madvr cannot identify hdr well (too much colour saturation and often red become the predominant colour), I think that the problem is madvr see that the input is hdr but cannot undertand the right color space.

![image](https://user-images.githubusercontent.com/84192597/118292812-4eed2780-b4d9-11eb-8dba-45cca93231b0.png)

I have tried to install python and the required extension but I cannot understand if they were working.

Thanks if you have any suggestions

**Comment by defl** [comment]
Sorry, totally missed this. Not yet used to looking at the issue tracker here. With the update you should now be able to get closer to your targer colorspace as the issue you're reporting was fixed.

As for the python bit, if it runs you can see the file's timestamp being updated every 10 seconds and see the values change as your input changes. If that all works then every time the filter starts up (stop-start) will load the then-correct values.

**Comment by Pinoginooo** [comment]
Don't worry and thx a lot for your work. I am sorry I opened this issue, but only after I discovered that my problem was not related to video_transfer_function: I was using avermedia card, and did not undarstand I was forcing hdr (injection filter) on a tone mapped stream (the avermedia do tone mapping when receives a hdr stream), I solved the issue using vertex in order  to say the card that the stream is not hdr. (the result was definitely really good when I applied madvr tone mapping) 

Now I am testing the Blackmagic card with your updated filter (vers 1.0, on Aver used only vers. 0.2), first thing I noted is that this card is much more sensible to refresh rate change than avermedia, but I made it run in some way (I have to force 29.98 refresh rate and set all the chain to this fixed refresh rate, while with Aver I easily used 23.976, and minor change does not create any problems with that card). 

The problem with this card is that I can run hdr with Madvr only in passthrough mode, if tone mapping option is set in madvr the result is a blackscreen or a fixed image. I have to test it more. 

Also I have to say that I didn't find any notable information increase in Madvr osd using this new  filter, If you are interested I will keep you informed on my findings.
